package.json

```diff
@@ -10,7 +10,8 @@
   ],
   "scripts": {
     "test": "mocha --harmony tests/**",
-    "test-frontend": "mocha --harmony examples/multiple-fragments-with-custom-amd/test.js",
+    "test-frontend": "mocha --harmony examples/fragments-performance/hooks.js",
```

Comment:
need to be `node examples/fragments-performance/hooks.js` 

Also rename it to `test:frontend:hooks`

Reply:
I have created a new file `examples/fragments-performance/test.js` instead and will run it via `mocha` to have a test spec instead of just running a node script